Scala/README.md

```diff
@@ -2,11 +2,27 @@
 
 ## Getting Started
 
-Before you start installing Scala on your machine, you must have Java 1.8 or greater installed on your computer.
+Before you start installing Scala on your machine, you must have Java 1.8 or greater installed on your computer. Type `java -version` to check the Java version.
+
+
 
 ### Installing Scala binaries
 
 * Windows: Install [Scala binaries](http://www.scala-lang.org/download/).
+* Linux: Install  
+&nbsp;&nbsp;&nbsp;&nbsp; - Ubuntu and other Debian-based distro  
+                   `echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list`  
+                    `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823`  
+                    `sudo apt-get update`  
+                    `sudo apt-get install sbt`  
+ &nbsp;&nbsp;&nbsp;&nbsp; - Red Hat Enterprise Linux and other RPM-based distro  
+                 `curl https://bintray.com/sbt/rpm/rpm | sudo tee/etc/yum.repos.d/bintray-sbt-rpm.repo`  
+                 `sudo yum install sbt`  
+* MacOS: Install  
+&nbsp;&nbsp;&nbsp;&nbsp;- [Using Homebrew](https://brew.sh/)
+                    `$ brew install sbt`  
+&nbsp;&nbsp;&nbsp;&nbsp;- [Using Macports](https://www.macports.org/)
```

Comment:
Remove `&nbsp;` Use `#` if you want to divide the article!

Reply:
thanks for the instructions... i will fix it again.